x11-themes/papirus-icon-theme/papirus-icon-theme-20210302.ebuild

```diff
@@ -0,0 +1,26 @@
+# Copyright 1999-2021 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+inherit gnome2-utils xdg-utils
+
+DESCRIPTION="icon theme for Linux"
+HOMEPAGE="https://github.com/PapirusDevelopmentTeam/papirus-icon-theme"
+SRC_URI="https://github.com/PapirusDevelopmentTeam/papirus-icon-theme/archive/${PV}.tar.gz -> ${P}.tar.gz"
+
+LICENSE="GPL-3"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+
+src_compile() { :; }
+
+src_install() {
+	insinto /usr/share/icons
+	doins -r ePapirus Papirus{,-Dark,-Light}
+
+}
+
+pkg_preinst() { gnome2_icon_savelist; }
+pkg_postinst() { gnome2_icon_cache_update; }
+pkg_postrm() { xdg_icon_cache_update; }
```

Comment:
Just inherit *xdg.eclass* and you can remove all of these phases from this ebuild.

Reply:
Thanks, I have also removed gnome2-utils from inherit

Reply:
Yes, you should only have `inherit xdg`. That's enough.